make.bat

```diff
@@ -72,6 +70,17 @@ if "%1" == "html" (
 	goto end
 )
 
+if "%1" == "htmlview" (
```

Comment:
This should be documented in the help above.

Reply:
Done!